griptape/tools/google_sheets/tool.py

```diff
@@ -0,0 +1,436 @@
+from __future__ import annotations
+import logging
+from schema import Schema, Literal, Optional, Or
+from attr import define, field
+from griptape.artifacts import (
+    ErrorArtifact,
+    InfoArtifact,
+    ListArtifact,
+    BlobArtifact,
+)
+from griptape.utils.decorators import activity
+from griptape.tools import BaseGoogleClient
+
+
+@define
+class GoogleSheetsClient(BaseGoogleClient):
+    SHEETS_SCOPES = ["https://www.googleapis.com/auth/spreadsheets"]
+
+    DRIVE_READ_SCOPES = ["https://www.googleapis.com/auth/drive.readonly"]
+
+    DRIVE_AUTH_SCOPES = ["https://www.googleapis.com/auth/drive"]
+
+    DRIVE_UPLOAD_SCOPES = ["https://www.googleapis.com/auth/drive.file"]
+
+    DEFAULT_FOLDER_PATH = "root"
```

Comment:
I think we can move all those (and the ones in other Google tools) into `BaseGoogleClient` since some of them are starting to repeat in different tools.

Reply:
I can move all of them except DEFAULT_FOLDER_PATH since it's being used in the activity.